languages/csharp/exercises/concept/enums/.docs/instructions.md

```diff
@@ -1,27 +1,36 @@
+# Instructions
+
 In this exercise you'll be processing log-lines.
 
-Each log line is a string formatted as follows: `"[<LEVEL>]: <MESSAGE>"`.
+Each log line is a string formatted as follows: `"[<LVL>]: <MESSAGE>"`.
 
-There are three different log levels:
+These are the different log levels:
 
-- `INFO`
-- `WARNING`
-- `ERROR`
+- `Unknown`
+- `Trace`
+- `Debug`
+- `Info`
+- `Warning`
+- `Error`
+- `Fatal`
 
 You have three tasks.
 
 ### 1. Parse log level
 
-Define a `LogLevel` enum that has three elements:
+Define a `LogLevel` enum that has seven elements. Here are the log levels and their corresponding messages from the system.
 
-- `Info`
-- `Warning`
-- `Error`
+- `Trace` - "[TRC] Here is the stack trace: ..."
+- `Debug` - "[DBG] Solve this problem by ..."
+- `Info` - "[INF] Processing..."
+- `Warning` - "[WRN] Unsuspicious activity detected"
+- `Error` - "[ERR] Missing semicolon;"
+- `Fatal` - "[FTL] Stack Overflow"
```

Comment:
With the log level format and the log levels already being defined above, the extra message is unneeded:

```suggestion
- `Trace`
- `Debug`
- `Info`
- `Warning`
- `Error`
- `Fatal`
```